.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
- - 5.5
```

Comment:
> PHPunit 5.2 does not support PHP 5.5 anymore.

[But we do](https://github.com/heiseonline/shariff-backend-php#requirements). I don't think it's a good idea to stop testing a supported version of PHP.

> That's ok for running the tests IMO.

PHP 5.5 ist also used to deploy the builds.